system/View/Cell.php

```diff
@@ -233,6 +233,8 @@ final protected function renderCell(BaseCell $instance, string $method, array $p
      * Returns the values from $params that match the parameters
      * for a method, in the order they are defined. This allows
      * them to be passed directly into the method.
+     *
+     * @return array
```

Comment:
This method is private so it is safe to apply the native return type.

Reply:
Ok.